addons/event_sale/tests/test_event_sale.py

```diff
@@ -376,7 +384,200 @@ def test_ticket_price_with_pricelist_and_tax(self):
             'event_id': event.id,
             'event_ticket_id': event_ticket.id,
         })
-        self.assertEqual(so.amount_total, 660.0, "Ticket is $1000 but the event product is on a pricelist 10 -> 6. So, $600 + a 10% tax.")
+        self.assertAlmostEqual(so.amount_total, 6.6, msg="Ticket is $1000 but the pricelist says fixed price $6 -> So, $6 + a 10% tax.")
+
+    def test_ticket_price_with_price_list_extensive(self):
+        """Extensive test of event ticket sales using price list.
+
+        Currently, multiple event ticket with various price can be linked to one product.
```

Comment:
Isn't `'Currently'` implying a future change, whether in the next lines or another commit? I wouldn't use it here.
```suggestion
        Multiple event tickets (with various prices) can be linked to a single product.
```

Reply:
Yes that's better, I have updated the comment.